### PR TITLE
oneIPL TAB December 16th MM

### PR DESCRIPTION
**Pull request by SvetlanaPodogova**
oneIPL TAB December 16th MM

### CODE REVIEW

File: tab-oneipl/minutes/2021_12_16_Minutes.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+
+oneIPL specification walk-through:
+
+* The latest version of oneIPL Spec is published on `oneIPL Spec
+  page`_.
+
+* oneAPI Image Processing Library (oneIPL) consists of several domains,
+  which includes Basic functionality, Color conversion operations, Filtering,
+  Geometry related functionality and 3D operations. Current version of
```
Comment (victorget) [inline tab-oneipl/minutes/2021_12_16_Minutes.rst]:
```suggestion
  Geometry related functionality and potentially can be extended to 3D operations. Current version of
```

File: tab-oneipl/minutes/2021_12_16_Minutes.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  specification covers the most important parts of domains and might be
+  extended in future versions.
+
+* The oneIPL programming language is SYCL 2020 based on C++17
+  oneIPL primitives include class data abstractions and functional API
+
+* ipl::image class specifies image data, layout and supported data types,
+  which are defined at compile-time. Each algorithm in the specification
+  contains the supported data types matrix. This matrix describes generic
```
Comment (victorget) [inline tab-oneipl/minutes/2021_12_16_Minutes.rst]:
```suggestion
  contains the layout and data type support matrix. This matrix describes generic
```

File: tab-oneipl/minutes/2021_12_16_Minutes.rst [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+* Robert Schneider, Siemens Healthineers: Are there query for HW feature like
+  format/type?
+  Victor Getmanskiy, Intel: Good question, we also see a gap here and consider
+  such extension to current SYCL images in future standards. Currently we can
+  query only if image/FP64/FP16 are supported or not.
+
+* Sergey Ivanov, OpenCV, Intel: You've said, the SYCL queue carries device
+  context - does all the objects need to belong to the same context?
+  Victor Getmanskiy, Intel: Yes, and each function has runtime checks that
```
Comment (victorget) [inline tab-oneipl/minutes/2021_12_16_Minutes.rst]:
```suggestion
  Victor Getmanskiy, Intel: Yes, and spec recommends for each function to have a runtime checks that
```

Comment (SvetlanaPodogova) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst]: thanks, Victor. Fixed

File: tab-oneipl/minutes/2021_12_16_Minutes.rst (hunk 2 above)
Comment (SvetlanaPodogova) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst]: fixed

File: tab-oneipl/minutes/2021_12_16_Minutes.rst (hunk 1 above)
Comment (SvetlanaPodogova) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst]: fixed

File: tab-oneipl/minutes/2021_12_16_Minutes.rst [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  shared, device and special texture memory. From implementation perspective
+  first three can be done via USM (SYCL2020), texture can be implemented
+  via sycl::image.
+
+* Memory model. oneapi::ipl::image class is basic data abstraction for image
+  data. oneIPL provides single abstraction over different memory types.
+  oneIPL supports different types of memory – device, host, shared and special
+  GPU texture (image) memory.
+
+* See more details on `oneIPL Architecture page`_
```
Comment (vkubarev) [inline tab-oneipl/minutes/2021_12_16_Minutes.rst:92]: Extra symbol at the end of line

Comment (SvetlanaPodogova) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst:92]: Dou you mean "page`_" ?
This is not extra symbol, this is format of links in rst syntax

File: tab-oneipl/minutes/2021_12_16_Minutes.rst [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+* Tim van der Horst, Philips: If the checks for the supported data types are
+  done before kernel, are they done in each call?
+  Victor Getmanskiy, Intel: Implementation can be done in a way to allow to
+  disable the checks to avoid performance overhead. In this case single initial
+  check is required in user Application.
+
+* Robert Schneider, Siemens Healthineers: Can the ipl::image be constructed
+  over sycl::image?
+  Victor Getmanskiy, Intel: Currently sycl::image 2020 is not supported by any
+  compiler, and SYCL image specification is improving, but in future based on
```
Comment (vkubarev) [inline tab-oneipl/minutes/2021_12_16_Minutes.rst]: SYCL image specification is being improved

Comment (SvetlanaPodogova) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst]: thanks, fixed

File: tab-oneipl/minutes/2021_12_16_Minutes.rst (hunk 4 above)
Comment (vkubarev) [reply tab-oneipl/minutes/2021_12_16_Minutes.rst:92]: ok, understand

**Review by vkubarev — APPROVED** [review]

**Review by victorget — APPROVED** [review]

**Pull request merged**